src/lib/Microsoft.Fx.Portability.Reports.Html/HtmlRazorReportWriter.cs

```diff
@@ -71,12 +74,57 @@ private static ServiceProvider BuildProvider()
 
             services.AddSingleton<ObjectPoolProvider, DefaultObjectPoolProvider>();
             services.AddLogging();
-            services.AddMvc();
-            services.AddTransient<RazorViewToStringRenderer>();
+            services
+                .AddMvc()
+                .ConfigureApplicationPartManager(partsManager => ConfigureRazorViews(partsManager, fileProvider));
 
+            services.AddTransient<RazorViewToStringRenderer>();
             return services.BuildServiceProvider();
         }
 
+        /// <summary>
+        /// Checks if the precompiled razor views have been added as an
+        /// ApplicationPart. If not, adds them.
+        /// On .NET 461, Microsoft.Fx.Portability.Reports.Html.dll
+        /// and Microsoft.Fx.Portability.Reports.Html.Views.dll are
```

Comment:
Is `Microsoft.Fx.Portability.Reports.Html.Views.dll` generated by the SDK?

Reply:
Yes. It's generated by the Razor SDK